### PR TITLE
Split tools

### PR DESCRIPTION
**Pull request by danieldunderfelt**
A cli for splitting style.json into parts. Also includes a command to render a combined style.json to stdout.

Discussion is needed to decide the grouping and evaluate if splitting by the source layer is a good thing to do.

### CODE REVIEW

File: .eslintrc.js [hunk 1]
```diff
@@ -2,6 +2,7 @@ module.exports = {
   extends: "airbnb-base/legacy",
   rules: {
     "quotes": ["error", "double", {"avoidEscape": true}],
+    "vars-on-top": "off"
```
Comment (danieldunderfelt) [inline .eslintrc.js]: The dumbest rule ever... jeez

File: .nvmrc [hunk 2]
```diff
@@ -0,0 +1 @@
+6
```
Comment (danieldunderfelt) [inline .nvmrc]: Require node 6 with nvm.

File: .eslintrc [hunk 3]
```diff
@@ -0,0 +1,15 @@
+{
+  "extends": ["airbnb-base", "plugin:prettier/recommended"],
+  "plugins": ["import"],
+  "rules": {
+    "quotes": ["error", "double", {"avoidEscape": true}],
+    "vars-on-top": "off",
```
Comment (hannesj) [inline .eslintrc]: This is not needed anymore

Comment (danieldunderfelt) [reply .eslintrc]: What do you mean? My editor was complaining about the rule, which I find ridiculous, so I disabled it.

**Pull request merged**